qa/workunits/rbd/rbd-ggate.sh

```diff
@@ -32,6 +32,19 @@ _sudo()
     sudo -nE "${cmd}" "$@"
 }
 
+check_geom_gate() 
+{
+    # See if geom_date is load, or can be loaded.
+    # Otherwise the tests can not run
+    if ! kldstat -q -n geom_gate ; then 
+        # See if we can load it
+        if ! _sudo kldload geom_gate ; then
+	    echo Not able to load geom_gate
+	    echo check /var/log/messages as to why 
```

Comment:
@wjwithagen Don't you want `return 1` or `exit 1` here so the script would not continue?

Reply:
@trociny 
Yup, that was there in the test.

Also added some more fixes, as the _sudo does not need to go on `rbd-ggate list` and that obfuscates the debug output only
Also `rbd resize` DOES require a _sudo(), so I added it.

And made some comments into echos so we can see what is tested.